python/utils/trace.py

```diff
@@ -265,6 +268,32 @@ def configure_simple_tracing_aie2(
     for addr, value in all_reg_writes.items():
         npu_write32(column=int(tile.col), row=int(tile.row), address=addr, value=value)
 
+
+def configure_broadcast_core_aie2(tile, num, event):
```

Comment:
This appears to be an unused function?

Reply:
Well, these are function wrappers. So I use them to wrap other more complex trace configs.